src/worksheet.h

```diff
@@ -24,5 +24,29 @@ class Worksheet : public Napi::ObjectWrap<Worksheet> {
   Napi::Value WriteNumber(const Napi::CallbackInfo& info);
   Napi::Value WriteString(const Napi::CallbackInfo& info);
   Napi::Value WriteURL(const Napi::CallbackInfo& info);
+  Napi::Value Autofilter(const Napi::CallbackInfo& info);
+  Napi::Value DataValidationCell(const Napi::CallbackInfo& info);
+  Napi::Value DataValidationRange(const Napi::CallbackInfo& info);
   lxw_worksheet* worksheet = nullptr;
 };
+
+class DataValidation {
```

Comment:
I'm not sure I understand, how to do it without copying strings. If I make a function like `lxw_data_validation toDataValidation(const Napi::Value& value)`, extract string values like `value.Get("errorText").As<Napi::String>().operator std::string().c_str()` and put them into `lxw_data_validation` instead that I then return, then these strings will be deallocated after the function ends, and so the caller of `toDataValidation` will get a struct with dangling pointers? What am I missing?

Reply:
I mean I could probably inline the code that transforms `Napi::Value` to `lxw_data_validation`, but it is needed in several worksheet methods, so not ideal. And won't I still need to save the strings `value.Get("errorText").As<Napi::String>().operator std::string()` to temporary variables, to make sure they don't get deallocated as temporary values?

Reply:
Yeah, inlining wouldn't be great. For now just remove it from `worksheet.h` and just keep the declaration and definition together in `worksheet.cc` since this is not meant to be exposed.